code/iaas/auth-logic/src/main/resources/META-INF/cattle/auth-logic/spring-iaas-auth-logic-context.xml

```diff
@@ -13,22 +13,27 @@
 		http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.0.xsd">
 
     <bean class="io.cattle.platform.iaas.api.auth.github.TokenResourceManager" >
-        <property name="githubTokenHandler">
-            <bean class="io.cattle.platform.iaas.api.auth.github.GithubTokenHandler"/>
-        </property>
+        <property name="tokenHandlers" value="#{extensionManager.list('token.handler')}" />
+
     </bean>
-    
+    <bean class="io.cattle.platform.iaas.api.auth.github.GithubTokenHandler" />
+
     <bean class="io.cattle.platform.iaas.api.auth.dao.impl.AuthDaoImpl" />
-	
+
     <bean id="BasicAuth" class="io.cattle.platform.iaas.api.auth.impl.BasicAuthImpl" />
     <bean class="io.cattle.platform.iaas.api.auth.impl.HeaderAuthLookup" >
         <property name="adminLookup" ref="BasicAuth" />
         <property name="adminAuthProvider" ref="DefaultAuthorizationProvider" />
     </bean>
-	
+
 	<bean class="io.cattle.platform.iaas.api.auth.github.ProjectResourceManager"/>
```

Comment:
Actually, wasn't referring to the lines that were changed.  Was looking at line 29 (from some previous commit).  And line 30 in new changes.  Those 2 lines look indented.  Also, line 37 in new file (once again from some previous commit).